File: Bab 7/7.8. CekKubus.java
import java.util.Scanner;

public class Main{

    public static void main(String[] args){

        int sisiPertama, sisiKedua, sisiKetiga;

        System.out.println("Masukan ketiga sisi");

        Scanner sc = new Scanner(System.in);

        try{

            sisiPertama = sc.nextInt();
            sisiKedua = sc.nextInt();
            sisiKetiga = sc.nextInt();

        }catch(Exception e){

        }

        if((sisiPertama == sisiKedua) && (sisiKedua == sisiKetiga)){

            System.out.println("Termasuk kubus");
    
        }else{

            System.out.println("Bukan termasuk kubus");
        }
    }
}

File: Bab 7/7.8. CekKubus.c
#include <stdio.h>

int main(){

    int sisiPertama, sisiKedua, sisiKetiga;

    printf("Masukan ketiga sisi\n");
    scanf("%d", &sisiPertama);
    scanf("%d", &sisiKedua);
    scanf("%d", &sisiKetiga);
    
    if((sisiPertama == sisiKedua) && (sisiKedua == sisiKetiga)){

        printf("Termasuk kubus\n");
    
    }else{

        printf("Bukan termasuk kubus\n");
    }

    return 0;
}